tensorflow/python/keras/losses.py

```diff
@@ -801,7 +801,12 @@ def categorical_hinge(y_true, y_pred):
   return math_ops.maximum(0., neg - pos + 1.)
 
 
+@keras_export('keras.losses.logloss')
```

Comment:
This is the same as binary crossentropy. Shouldn't we remove all instances of "logloss" from the API? (preferring binary_crossentropy instead).

Reply:
Good point. I removed the logloss function and class entirely since I don't think we want to keep those. Let me know if you want any more changes! 😄

Reply:
https://github.com/tensorflow/tensorflow/commit/55240a22f10a9a6cf3cf8ba3d35e16d88f5925f6#diff-b789cb2a4dba80832969100ce22623cb removed logloss. Please sync to the latest changes.

Reply:
Ok - synced with latest.